### PR TITLE
Including matching id by name for the security group name on rules param section

### PR DESCRIPTION
**Pull request by ccamacho**
This commit adds the parameter security_group_name
to the params section.

Closes: #79

### CODE REVIEW

**Comment by ccamacho** [comment]
To discuss the remote security group name

File: os_migrate/plugins/module_utils/network.py [hunk 1]
```diff
@@ -136,6 +136,27 @@ def network_refs_from_sdk(conn, sdk_net):
...
+    """
+    expected_type = openstack.network.v2.security_group_rule.SecurityGroupRule
+    if type(sdk_rule) != expected_type:
+        raise exc.UnexpectedResourceType(expected_type, type(sdk_rule))
+    refs = {}
+
+    # when creating refs from SDK Security Group Rule object, we copy IDs and
+    # query the cloud for names
+    refs['security_group_name'] = reference.security_group_name(
+        conn, sdk_rule['security_group_id'])
```
Comment (jistr) [inline os_migrate/plugins/module_utils/network.py]: We should add remote_group_name from remote_group_id, using the same security_group_name lookup function.

File: os_migrate/plugins/module_utils/network.py [hunk 2]
```diff
@@ -191,7 +212,10 @@ def serialize_security_group_rule(sdk_sec_rule):
         'revision_number',
     ])
 
-    # FIXME: missing remote_group_name and security_group_name
+    set_ser_params_same_name(params, sec_refs, [
+        'security_group_name',
```
Comment (jistr) [inline os_migrate/plugins/module_utils/network.py]: Ditto re adding remote_group_name

File: os_migrate/tests/unit/test_security_group_rule.py [hunk 3]
```diff
@@ -12,11 +12,13 @@ class TestSecurityGroupRule(unittest.TestCase):
...
     def test_serialize_security_group_rule(self):
         sec = fixtures.sdk_security_group_rule()
-        serialized = network.serialize_security_group_rule(sec)
+        sec_refs = fixtures.security_group_rule_refs()
+        serialized = network.serialize_security_group_rule(sec, sec_refs)
         s_params = serialized['params']
         s_info = serialized['_info']
 
         self.assertEqual(serialized['type'], 'openstack.network.SecurityGroupRule')
+        self.assertEqual(s_params['security_group_name'], 'default')
```
Comment (jistr) [inline os_migrate/tests/unit/test_security_group_rule.py]: Ditto re adding remote_group_name

File: os_migrate/plugins/module_utils/network.py (hunk 1 above)
Comment (ccamacho) [reply os_migrate/plugins/module_utils/network.py]: done

File: os_migrate/plugins/module_utils/network.py (hunk 2 above)
Comment (ccamacho) [reply os_migrate/plugins/module_utils/network.py]: done

**Pull request merged**